dmapiclient/data.py

```diff
@@ -248,6 +248,13 @@ def find_framework_suppliers(self, framework_slug, agreement_returned=None):
             params=params
         )
 
+    def put_signed_agreement_on_hold(self, framework_agreement_id, user):
```

Comment:
Would have been hard to spot but we have started a new section for framework agreement routes towards the bottom of the file (around line 650). This should be grouped with those please